PlexRequests.UI/Views/Shared/Partial/_Navbar.cshtml

```diff
@@ -91,5 +93,22 @@
             </ul>
         </div>
     </div>
+    <script>
+        $.ajax({
+            url: "/customDonation",
```

Comment:
We will have to make sure the `url` also includes the baseUrl. Take a look at some of the existing Javascript that does this (e.g. `SonarrSettings.cshtml` view)